app/models/content_item.rb

```diff
@@ -59,6 +59,14 @@ def base_path
     content_item_hash.dig('base_path')
   end
 
+  def raw_facets
+    @raw_facets ||= FacetExtractor.new(content_item_hash).extract
```

Comment:
What are raw facets here?

Reply:
Oh that is a refactor too many, I'll remove